modules/gke-cluster-standard/README.md

```diff
@@ -55,7 +55,7 @@ module "cluster-1" {
 # tftest modules=1 resources=1 inventory=basic.yaml
 ```
 
-### Enable Dataplane V2
+### Enable Dataplane
```

Comment:
Why remove the "V2" here?

Reply:
tfdoc.py removed it

Reply:
Are you sure it was tfdoc?

Reply:
Sorry, I didn't know what happened, I relaunched tfdoc.py and it worked properly.